Change Person to Student

tensaida opened the pull request:
The codebase contains mentions of "person", which is a remnant of the AB3 codebase.

Let's refactor mentions of person to student.

Fixes #42

tensaida committed:
Change Person to Student

The codebase contains mentions of "person", which is a remnant of the AB3 codebase.

Let's refactor mentions of person to student.

tensaida committed:
Change star import to single class import

It fails checkstyle checks.

Let's refactor the star imports to single clas imports.

LuYiting0913 reviewed (commented):
code under src/main seems ok

LuYiting0913 reviewed (commented):
docs looks fine

tensaida committed:
Change unit test method names to camelCase

The test code fails checkstyle checks.

Let's refactor unit test method names to camelCase as indicated in the checkstyle.

tensaida committed:
Fix test data

Some test cases failed.

Let's refactor "persons" to "students" in the json test data files.
